testsuite/tests/lib-unix/unix-socket/recvfrom_linux.ml

```diff
@@ -0,0 +1,20 @@
+(* TEST
+include unix
+modules = "recvfrom.ml"
+script = "sh ${test_source_directory}/is-linux.sh"
+* hasunix
+** script
+*** bytecode
+*** native
+*)
+open Recvfrom
+
+let () =
+  let server_path = "/tmp/ocaml-test-socket-linux" in
```

Comment:
We should probably not hard-code `/tmp` here. I suggest to just use a name relative to the current directory.